classes/Zipcode.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Value type wrapping validation logic for zipcode.
+ */
+class Zipcode {
+    public static $ZipcodeValidationRegex = '/^[0-9]{4}[a-zA-Z]{2}$/';
+    var $code;
+
+    public function __ctor($code) {
+        // Disabled for now, don't want to cause a 'merge' conflict with upcoming validation changes yet.
+        // if(!self::IsValid($code)){
+        //     throw new InvalidArgumentException('Code should be 4 numbers + 2 letters!');
+        // }
+        $this->code = $code;
+    }
+
+    public function getCode(){
+        return $this->code;
+    }
+
+    /**
+     * Mere stub, Wether this should really belong here can be questioned. But wanted to
+     * prevent HeldNodig becoming even an bigger monolith.
+     */
+    public function ToCity(){
```

Comment:
For the other methods too:
- start method name with lowercase letter
- please watch for spacing issues, so in this case `toCity() {`
- try to add types of the expected input and output, so `methodName(string $argument, bool $isOn, NameOfObject $someObject): string {` (and it returns a string  in this example

Reply:
Sorry, coming from C# so upper CamelCasing is a habit.

I didn't know php also supports typehint for primitives. Will do!